source/game/kart/KartParam.cc

```diff
@@ -34,6 +34,10 @@ bool KartParam::isBike() const {
     return m_isBike;
 }
 
+bool KartParam::isQuacker() const {
```

Comment:
This function should have a better name. `isQuacker` implies we're checking against the vehicle ID, not the body enum.